build-logic/build.gradle.kts

```diff
@@ -45,3 +45,7 @@ tasks.register("codeChecks").configure {
   description = "Runs code style and license checks"
   dependsOn("spotlessCheck")
 }
+
+kotlin {
```

Comment:
Gradle complains about Java 23 vs Kotlin targeting 22 max (and going to fail in Gradle 9 in such a case).